src/pages/Search/SearchFiltersDatePage.tsx

```diff
@@ -1,16 +1,40 @@
+import {format} from 'date-fns';
 import React from 'react';
 import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
 import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import DatePicker from '@components/DatePicker';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormOnyxValues} from '@components/Form/types';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import Text from '@src/components/Text';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Search from '@userActions/Search';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import INPUT_IDS from '@src/types/form/SearchAdvancedFiltersForm';
+
+const currentDate = format(new Date(), CONST.DATE.FNS_FORMAT_STRING);
```

Comment:
rename to `defaultDate`

Reply:
btw I wanted to point out a tricky thing which you would have to be careful for:
this call to `new Date()` will be run just once when this file is loaded for the first time, and then the date will be unchanged.
Since we're operating on days here its not a big problem. But if this would be a date+time picker, and we would operate on let's say hours, then if someone enters the app and doesn't close the webpage for more than an hour, he would have old date here.

But I think for us and for days it should be fine

Reply:
Thanks for the tip, it's worth remembering for the future :)